x-pack/plugins/observability/public/pages/slo_edit/slo_edit.test.tsx

```diff
@@ -108,9 +109,13 @@ const mockKibana = () => {
         get: () => {},
       },
       triggersActionsUi: {
-        getAddRuleFlyout: jest
-          .fn()
-          .mockReturnValue(<div data-test-subj="add-rule-flyout">Add Rule Flyout</div>),
+        getAddRuleFlyout: jest.fn().mockReturnValue(
```

Comment:
@elastic/actionable-observability 
I'm not sure how this end up in my PR, and is actually creating a conflict